src/Spice86.Core/Emulator/Function/Dump/ExecutionFlowDumper.cs

```diff
@@ -3,7 +3,7 @@
 using System.Diagnostics;
 using System.IO;
 
-using Newtonsoft.Json;
+using System.Text.Json;
```

Comment:
If STJ is *reliable* (ie. doesn't silently fail serialization with an empty JSON object as output, or deserialization with 'null' as ouput), we can keep it.

Reply:
Would have kept Newtonsoft but I couldnt make it work with SegmentedAddress having 2 constructors even after specifying [JsonConstructor]